download.md

```diff
@@ -6,6 +6,53 @@ show_hero: true
 ---
 
 <div class="container">
+   <!-- Card deck -->
+    <div class="card-deck spacer-30">
+    <div class="card mb-4">
```

Comment:
is it really mb-4 .. not md-4 ?

Reply:
I'm just following suite to the other cards just beneath.

Reply:
This is right